notebooks/examples/python/Mosaic + Sedona Raster 1.2.x Example.py

```diff
@@ -0,0 +1,731 @@
+# Databricks notebook source
+# MAGIC %md # Mosaic + Apache Sedona Raster 1.2.x Example
+# MAGIC 
+# MAGIC * Assumes DBR 10.4 which is Spark 3.2.1
+
+# COMMAND ----------
+
+# MAGIC %md _Example Cluster Config from demo shard (doesn't need to be single node)_
+# MAGIC ```
+# MAGIC {
+# MAGIC     "num_workers": 0,
+# MAGIC     "cluster_name": "mjohns-mosaic-sedona-sn",
+# MAGIC     "spark_version": "10.4.x-scala2.12",
+# MAGIC     "spark_conf": {
+# MAGIC         "spark.databricks.cluster.profile": "singleNode",
+# MAGIC         "spark.sql.extensions": "org.apache.sedona.viz.sql.SedonaVizExtensions,org.apache.sedona.sql.SedonaSqlExtensions",
+# MAGIC         "spark.master": "local[*, 4]",
+# MAGIC         "spark.kryo.registrator": "org.apache.sedona.core.serde.SedonaKryoRegistrator",
+# MAGIC         "spark.serializer": "org.apache.spark.serializer.KryoSerializer"
+# MAGIC     },
+# MAGIC     "aws_attributes": {
+# MAGIC         "first_on_demand": 1,
+# MAGIC         "availability": "SPOT_WITH_FALLBACK",
+# MAGIC         "zone_id": "us-west-2c",
+# MAGIC         "spot_bid_price_percent": 100,
+# MAGIC         "ebs_volume_count": 0
+# MAGIC     },
+# MAGIC     "node_type_id": "i3.xlarge",
+# MAGIC     "driver_node_type_id": "i3.xlarge",
+# MAGIC     "ssh_public_keys": [],
+# MAGIC     "custom_tags": {
+# MAGIC         "ResourceClass": "SingleNode"
+# MAGIC     },
+# MAGIC     "spark_env_vars": {
+# MAGIC         "PYSPARK_PYTHON": "/databricks/python3/bin/python3"
+# MAGIC     },
+# MAGIC     "autotermination_minutes": 60,
+# MAGIC     "enable_elastic_disk": true,
+# MAGIC     "cluster_source": "UI",
+# MAGIC     "init_scripts": [
+# MAGIC         {
+# MAGIC             "dbfs": {
+# MAGIC                 "destination": "dbfs:/FileStore/sedona/sedona-1.2.1-init-master.sh"
+# MAGIC             }
+# MAGIC         }
+# MAGIC     ],
+# MAGIC     "runtime_engine": "STANDARD",
+# MAGIC     "cluster_id": "0704-014006-t9f0pvs3"
+# MAGIC }
+# MAGIC ```
+
+# COMMAND ----------
+
+# MAGIC %md _For DBR, install python bindings_
+# MAGIC 
+# MAGIC > Instead of `%pip install apache-sedona==1.2.0` using WHL built from Sedona master (1.2.1)
+
+# COMMAND ----------
+
+# MAGIC %pip install /dbfs/FileStore/jars/sedona/1.2.1-incubating-master/apache_sedona-1.2.0-py3-none-any.whl
```

Comment:
This file does not exist by default, so in order to make it run out-of-the-box we should either use pypi or download this whl from a previous notebook cell.